src/manifest_parser_perftest.cc

```diff
@@ -115,4 +125,6 @@ int main(int argc, char* argv[]) {
   int max = *max_element(times.begin(), times.end());
   float total = accumulate(times.begin(), times.end(), 0.0f);
   printf("min %dms  max %dms  avg %.1fms\n", min, max, total / times.size());
+
+  return 0;
```

Comment:
Does wmain require this? If not, you don't need to add it.

Reply:
Removing it should only leave a warning when compiling using GCC. I can remove it

Reply:
GCC on Windows i.e. MinGW?

Reply:
The MinGW version.

Reply:
Leave it then :)